Sources/AegithalosCocoa/UIView+Setup.swift

```diff
@@ -118,126 +118,258 @@ public extension Setup where Subject: UIView {
-    equalTo anchor: NSLayoutAnchor<NSLayoutXAxisAnchor>,
+    _ relation: LayoutConstraintRelation,
+    _ anchor: NSLayoutAnchor<NSLayoutXAxisAnchor>,
     constant: CGFloat = 0,
     priority: UILayoutPriority = .required
   ) -> Setup {
     composed { [unowned anchor] (subject: Subject) in
       subject.translatesAutoresizingMaskIntoConstraints = false
-      let constraint = subject.centerXAnchor.constraint(equalTo: anchor, constant: constant)
+      let constraint: NSLayoutConstraint
+      switch relation {
+      case .equalTo:
+        constraint = subject.centerXAnchor.constraint(equalTo: anchor, constant: constant)
+      case .lessThanOrEqualTo:
+        constraint = subject.centerXAnchor.constraint(lessThanOrEqualTo: anchor, constant: constant)
+      case .greaterThanOrEqualTo:
+        constraint = subject.centerXAnchor.constraint(greaterThanOrEqualTo: anchor, constant: constant)
+      }
+      constraint.priority = priority
+      constraint.isActive = true
+    }
+  }
+  
+  @inlinable func width(
+    _ relation: LayoutConstraintRelation = .equalTo,
+    _ constant: CGFloat,
+    priority: UILayoutPriority = .required
+  ) -> Setup {
+    composed { (subject: Subject) in
+      let constraint: NSLayoutConstraint
+      switch relation {
+      case .equalTo:
+        constraint = subject.widthAnchor.constraint(equalToConstant: constant)
+      case .lessThanOrEqualTo:
+        constraint = subject.widthAnchor.constraint(lessThanOrEqualToConstant: constant)
+      case .greaterThanOrEqualTo:
+        constraint = subject.widthAnchor.constraint(greaterThanOrEqualToConstant: constant)
+      }
+      constraint.priority = priority
+      constraint.isActive = true
+    }
+  }
+  
+  @inlinable func height(
+    _ relation: LayoutConstraintRelation = .equalTo,
+    _ constant: CGFloat,
+    priority: UILayoutPriority = .required
+  ) -> Setup {
+    composed { (subject: Subject) in
+      let constraint: NSLayoutConstraint
+      switch relation {
+      case .equalTo:
+        constraint = subject.heightAnchor.constraint(equalToConstant: constant)
+      case .lessThanOrEqualTo:
+        constraint = subject.heightAnchor.constraint(lessThanOrEqualToConstant: constant)
+      case .greaterThanOrEqualTo:
+        constraint = subject.heightAnchor.constraint(greaterThanOrEqualToConstant: constant)
+      }
       constraint.priority = priority
       constraint.isActive = true
     }
   }
   
   @inlinable func width(
-    equalTo constant: CGFloat,
+    _ relation: LayoutConstraintRelation,
+    _ anchor: NSLayoutAnchor<NSLayoutDimension>,
+    constant: CGFloat = 0,
     priority: UILayoutPriority = .required
   ) -> Setup {
     composed { (subject: Subject) in
-      let constraint = subject.widthAnchor.constraint(equalToConstant: constant)
+      let constraint: NSLayoutConstraint
+      switch relation {
+      case .equalTo:
+        constraint = subject.widthAnchor.constraint(equalTo: anchor, constant: constant)
```

Comment:
Shouldn't it be as above: constraint = subject.widthAnchor.constraint(equalToConstant: constant)